api/v1alpha3/machinedeployment_types.go

```diff
@@ -157,7 +157,7 @@ type MachineRollingUpdateDeployment struct {
 type MachineDeploymentStatus struct {
 	// The generation observed by the deployment controller.
 	// +optional
```

Comment:
Should this be no longer optional, since we are not setting omitempty? https://github.com/eBay/Kubernetes/blob/master/docs/devel/api-conventions.md#optional-vs-required